packages/grafana-ui/src/components/Table/TableNG/Cells/AutoCell.tsx

```diff
@@ -15,14 +16,22 @@ export function AutoCell({ value, field, rowIdx }: AutoCellProps) {
   );
 }
 
-export const getStyles: TableCellStyles = (_theme, { textWrap, shouldOverflow }) =>
+export const getStyles: TableCellStyles = (_theme, { textWrap, shouldOverflow, maxHeight }) =>
   css({
     ...(textWrap && { whiteSpace: 'pre-line' }),
     ...(shouldOverflow && {
       '&:hover, &[aria-selected=true]': {
         whiteSpace: 'pre-line',
       },
     }),
+    ...(typeof maxHeight === 'number' && {
+      height: 'auto',
+      minHeight: 'none',
+      overflowY: 'hidden',
+      display: '-webkit-box',
+      WebkitBoxOrient: 'vertical',
+      WebkitLineClamp: Math.floor(maxHeight / TABLE.LINE_HEIGHT),
```

Comment:
- https://developer.mozilla.org/en-US/docs/Web/CSS/line-clamp
- https://caniuse.com/?search=line-clamp

the non-vendored `line-clamp` isn't supported widely, and `-webkit-line-clamp` is. I found when I tried to use the unvendored version that it didn't work.